playbooks/roles/suse-build-images/tasks/main.yml

```diff
@@ -20,91 +26,15 @@
   tags:
     - install
 
-- name: Ensure docker is running on host
-  service:
-    name: docker
-    state: started
-
-- name: Check if local docker registry server is running
-  shell: |
-    set -o pipefail
-    docker ps | grep {{ local_registry_name }}
-  register: registry_running_cmd_status
-  changed_when: false
-  failed_when: false
-  tags:
-    - run
-
-# TODO(evrardjp) move this to docker module and remove skip_ansible_lint
-- name: Start local docker registry if not running
-  shell: |
-    docker run -d -p {{ local_registry_port }}:{{ local_registry_port }} --name {{ local_registry_name }} --restart always registry:2
-  when:
-    - registry_running_cmd_status.rc != 0
-  tags:
-    - skip_ansible_lint
-
-- name: Ensure local-registry DNS resolution is set in /etc/hosts
-  blockinfile:
-    path: /etc/hosts
-    block: |
-      {{ ansible_default_ipv4.address | ipaddr('address') }} {{ local_registry_name }}
-
-- name: Make sure docker daemon file is present on host
-  copy:
-    dest: "/etc/docker/daemon.json"
-    content: "{}"
-    force: no
-    mode: 0644
-
-- name: importing the file contents to variable
-  slurp:
-    src: /etc/docker/daemon.json
-  register: _docker_daemon_conf_json
-  become: true
-
-- name: Read existing docker daemon conf file
-  set_fact:
-     _docker_daemon_conf: "{{ (_docker_daemon_conf_json.content | b64decode | from_json) }}"
-
-- name: Set registry entry in docker daemon conf if not defined
-  set_fact:
-    _upd_docker_daemon_conf: "{{ _docker_daemon_conf | combine({ item.key: item.value }) }}"
-  with_items:
-    - { key: "insecure-registries", value: "[\"{{ local_registry_name }}:{{ local_registry_port }}\"]" }
-  when:
-    - _docker_daemon_conf | json_query('"insecure-registries"') | default("undefined", true) == "undefined"
-
-- name: Write back updated docker daemon file
-  copy:
-    content: "{{ _upd_docker_daemon_conf | to_nice_json }}"
-    dest: "/etc/docker/daemon.json"
-  when:
-    - _upd_docker_daemon_conf is defined
-
-- name: Restart docker if needed
-  service:
-    name: docker
-    state: restarted
-  when: _upd_docker_daemon_conf is defined
-
-- name: Build infra components
-  include_tasks: build-infra-component.yml
-  with_items: "{{ build_infra_components_list }}"
+- name: Build and push the non-LOCI components
```

Comment:
Wouldn't it be good to mention that those are actually openstack-helm-images

Reply:
They are all openstack-helm-images?

Reply:
I mean: unlike LOC components, these are coming from different source, openstack-helm-images project

Reply:
Solved in PR96.